pkg/test/dependency/dependencies.go

```diff
@@ -18,8 +18,11 @@ const (
 	// Apps is a dependency on fake networked apps. This can be used to mimic traffic thru a mesh.
 	Apps = Instance("apps")
 
+	// Cluster is required for running the test.
+	Cluster = Instance("cluster")
+
 	// GKE dependency
-	GKE = Instance("cluster")
+	GKE = Instance("gke")
```

Comment:
Does `GKE` imply `Cluster`? What do we do if a test depends on both `Cluster` and `GKE`? I assume that doesn't result in 2 environments being created?